apis/kueue/webhooks/workload_webhook.go

```diff
@@ -76,13 +126,31 @@ func setContainersDefaults(containers []corev1.Container) {
 			if c.Resources.Requests == nil {
 				c.Resources.Requests = make(corev1.ResourceList)
 			}
-			for k, v := range c.Resources.Limits {
-				if _, exists := c.Resources.Requests[k]; !exists {
-					c.Resources.Requests[k] = v.DeepCopy()
-				}
-			}
+			c.Resources.Requests = mergeResourceList(c.Resources.Requests, c.Resources.Limits)
+		}
+	}
+}
+
+func setContainersDefaultsFromLimitRanges(containers []corev1.Container, limits []corev1.LimitRangeItem) {
+	for _, ri := range limits {
+		for i := range containers {
+			cr := &containers[i].Resources
+			cr.Limits = mergeResourceList(cr.Limits, ri.Default)
+			cr.Requests = mergeResourceList(cr.Requests, ri.DefaultRequest)
```

Comment:
what about Min and Max?

Reply:
In my opinion Min and Max  should only be used in validation.